source/manual/emergency-publishing.html.md

```diff
@@ -7,172 +7,182 @@ section: Publishing
-- [https://www.gov.uk/search?q=69b197b8](https://www.gov.uk/search?q=69b197b8) ([Integration](https://www-origin.integration.publishing.service.gov.uk/search?q=69b197b8), [Staging](https://www-origin.staging.publishing.service.gov.uk/search?q=69b197b8))
+If the banner doesn't show, see [Troubleshoot the emergency banner](#troubleshoot-the-emergency-banner).
 
-If the banner doesn't show, [look at the troubleshooting section](#the-banner-is-not-showing--not-clearing).
+#### Some example pages to check
 
-Once all caches have had time to clear, check that the emergency banner is visible when accessing the same pages as before but without a cache-bust string.
+You must change the cache-bust strings in these examples in order for them to
+be accurate tests of the banner deployment.
 
-- [https://www.gov.uk/](https://www.gov.uk/) ([Integration](https://www-origin.integration.publishing.service.gov.uk/), [Staging](https://www-origin.staging.publishing.service.gov.uk/))
-- [https://www.gov.uk/financial-help-disabled](https://www.gov.uk/financial-help-disabled) ([Integration](https://www-origin.integration.publishing.service.gov.uk/financial-help-disabled), [Staging](https://www-origin.staging.publishing.service.gov.uk/financial-help-disabled))
-- [https://www.gov.uk/government/organisations/hm-revenue-customs](https://www.gov.uk/government/organisations/hm-revenue-customs) ([Integration](https://www-origin.integration.publishing.service.gov.uk/government/organisations/hm-revenue-customs), [Staging](https://www-origin.staging.publishing.service.gov.uk/government/organisations/hm-revenue-customs))
-- [https://www.gov.uk/search](https://www.gov.uk/search) ([Integration](https://www-origin.integration.publishing.service.gov.uk/search), [Staging](https://www-origin.staging.publishing.service.gov.uk/search))
+- [https://www.gov.uk/?ae00e491](https://www.gov.uk/?ae00e491) ([staging](https://www-origin.staging.publishing.service.gov.uk/?ae00e491))
+- [https://www.gov.uk/financial-help-disabled?7f7992eb](https://www.gov.uk/financial-help-disabled?7f7992eb) ([staging](https://www-origin.staging.publishing.service.gov.uk/financial-help-disabled?7f7992eb))
+- [https://www.gov.uk/government/organisations/hm-revenue-customs?49854527](https://www.gov.uk/government/organisations/hm-revenue-customs?49854527) ([staging](https://www-origin.staging.publishing.service.gov.uk/government/organisations/hm-revenue-customs?49854527))
+- [https://www.gov.uk/search?q=69b197b8](https://www.gov.uk/search?q=69b197b8) ([staging](https://www-origin.staging.publishing.service.gov.uk/search?q=69b197b8))
```

Comment:
I would just omit the cachebust strings. If we're having to change them anyway, having them there isn't very helpful, and we already explain the need to add cachebust strings in the section above this one.

Reply:
Yeah was thinking the same thing. Done :)